websites/S/Stremio/presence.ts

```diff
@@ -3,172 +3,404 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
-let timestamp: [number, number],
-	pauseCheck: boolean,
-	search: HTMLInputElement,
-	title: string;
+enum AppVersion {
+	Website = -1,
+	V4 = 4,
+	V5 = 5,
+}
+
+const enum Assets {
+	Logo = "https://cdn.rcd.gg/PreMiD/websites/S/Stremio/assets/logo.png",
+}
+
+function getApVersion(hostname: string) {
```

Comment:
Lil typo, make sure to change it everywhere tho and not just accept my suggestion
```suggestion
function getAppVersion(hostname: string) {
```

Reply:
Nice catch! Fixed.